--- dbus-java-2.3.1/Makefile.orig	2007-07-26 13:12:33.000000000 +0200
+++ dbus-java-2.3.1/Makefile	2007-08-12 13:00:01.976601441 +0200
@@ -61,7 +61,7 @@
 	mkdir -p classes
 	$(JAVAC) -cp classes:${JAVAUNIXJARDIR}/debug-$(DEBUG).jar:${JAVAUNIXJARDIR}/hexdump.jar:$(CLASSPATH) -d classes $(JCFLAGS) $(SRCDIR)/dbus/test/*.java
 	touch .testclasses 
-.viewerclasses: $(SRCDIR)/dbus/viewer/*.java .classes
+.viewerclasses: $(SRCDIR)/dbus/viewer/*.java .binclasses
 	mkdir -p classes
 	$(JAVAC) -cp classes:$(CLASSPATH):${JAVAUNIXJARDIR}/unix.jar:${JAVAUNIXJARDIR}/debug-$(DEBUG).jar:${JAVAUNIXJARDIR}/hexdump.jar -d classes $(JCFLAGS) $(SRCDIR)/dbus/viewer/*.java
 	touch .viewerclasses 
